### ISSUE TITLE
PermissionDenied when trying to create resources on self hosted instance.

### ISSUE THREAD

**Issue by rubenelshof** [zitadel/terraform-provider-zitadel#199]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Version

2.0.1

### ZITADEL Version

2.62.1

### Describe the problem caused by this bug

I am trying to create an organization.
After running `tofu apply` I receive the following error.
```
zitadel_org.homelab: Creating...
╷
│ Error: rpc error: code = PermissionDenied desc = could not read projectid by clientid (AUTH-GHpw2)
│ 
│   with zitadel_org.homelab,
│   on org_homelab.tf line 1, in resource "zitadel_org" "homelab":
│    1: resource "zitadel_org" "homelab" {
│ 
╵
```

### To reproduce

I have followed the steps on the official docs website to setup a [Docker compose with service account](https://zitadel.com/docs/self-hosting/deploy/compose#docker-compose-with-service-account).
I copied the `zitadel-admin-sa.json` key to my laptop from which I run OpenTofu.
My terraform config
```
terraform {
  required_providers {
    zitadel = {
      source  = "zitadel/zitadel"
      version = "2.0.1"
    }
  }
}

provider "zitadel" {
  domain           = "foo.bar"
  insecure         = "false"
  port             = "443"
  jwt_profile_file = "./zitadel-admin-sa.json"
}

resource "zitadel_org" "homelab" {
  name       = "homelab"
  is_default = true
}
```

**Comment by joeppeeters** [comment]
We're experiencing the same issue. Our workaround for now is to downgrade to [v2.61.1](https://github.com/zitadel/zitadel/releases/tag/v2.61.1).

**Comment by rubenelshof** [comment]
Can confirm that the workaround of downgrading to v2.61.1 works.

**Comment by rubenelshof** [comment]
The issue was fixed for me in the latest version [v2.63.1](https://github.com/zitadel/zitadel/releases/tag/v2.63.1)
